src/custom/pages/NewSwap/index.tsx

```diff
@@ -0,0 +1,25 @@
+import * as styledEl from './styled'
+import { GNO_GNOSIS_CHAIN, WETH_GNOSIS_CHAIN } from 'utils/gnosis_chain/constants'
+import { ReceiveAmount } from './pureComponents/ReceiveAmount'
+import { CurrencyInputPanel } from './pureComponents/CurrencyInputPanel'
+import { CurrencyArrowSeparator } from './pureComponents/CurrencyArrowSeparator'
+import { TradeRates } from './pureComponents/TradeRates'
+import { TradeButton } from './pureComponents/TradeButton'
+import { useDerivedSwapInfo } from 'state/swap/hooks'
+
+export function NewSwapPage() {
+  const { allowedSlippage } = useDerivedSwapInfo()
+
+  return (
+    <styledEl.Container>
```

Comment:
shouldn't the swap widget composition be inside a component instead of the page?

Reply:
I moved it because `AffiliateStatusCheck` component is out of boundaries